### ISSUE TITLE
508-defect-4 [COGNITION]: App Directory - Links with same name should have similar purposes

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16450]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a user who might choose to expand more than one `Additional Information` component, I would like the links to have unique text so I can tell what app each link is referring to.

## Details

We have three links under each app's additional information pane that have the same text but unique URLs:

* View the privacy policy
* View terms of service
* Report this app to VA

It would help us strive for WCAG2 AAA (beyond compliance) if these links had the app's name, so users could distinguish them without additional context like headings.

## Acceptance Criteria

- [ ] Axe "Links with the same name have a similar purpose" best practice warning no longer appears in future scans
- [ ] No new axe browser violations

## Solution (if known)

```diff
<a
  class="vads-u-margin-top--2"
  href="https://www.apple.com/legal/privacy/"
  rel="noopener noreferrer"
  target="_blank"
- >View the privacy policy</a>
+ >View the Apple Health privacy policy</a>
```

```diff
<a
  class="vads-u-margin-top--1"
  href="https://www.apple.com/legal/sla/"
  rel="noopener noreferrer"
  target="_blank"
- >View terms of service</a>
+ >View Apple Health terms of service</a>
```

```diff
<a
  class="vads-u-margin-top--1"
  href="mailto:api@va.gov?subject=Report Apple Health to VA"
  rel="noopener noreferrer"
  target="_blank"
- >Report this app to VA</a>
+ >Report Apple Health to VA</a>
```

## WCAG or Vendor Guidance (optional)

* [Link Purpose (Link Only): Understanding SC 2.4.9](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html)

## Screenshots or Trace Logs

![Screen Shot 2020-11-19 at 3 55 51 PM](https://user-images.githubusercontent.com/934879/99732861-dd7e7000-2a85-11eb-8d42-6b7cc81301c2.png)

**Comment by LeKeve** [comment]
Ready to be tested in production.

**Comment by 1Copenut** [comment]
LGTM. Moving to close.
